core/collector.class.inc.php

```diff
@@ -636,7 +636,15 @@ protected function RemoveDataFiles()
 		}
 	}
 
-	public function Synchronize($iMaxChunkSize = 0)
+
+	/**
+	 * @param bool $bSynchroDetails
+	 * @param $iMaxChunkSize
+	 *
+	 * @return bool
+	 * @throws \Exception
+	 */
+	public function Synchronize(bool $bSynchroDetails, $iMaxChunkSize = 0)
```

Comment:
Discussed during collector PRs review.

As Anne-Catherine pointed out, this method is overloaded by other collectors, so changing its prototype by inserting an argument first will break derived classes.

Also we would like to keep the consistency of debugging by keeping this kind of output in the logs. So we would tend to using `Utils::Log` with a DEBUG level.